cpp/cmake/config.json

```diff
@@ -38,5 +46,8 @@
     "public_var_pattern": "[A-z][0-9A-z_]+",
     "argument_var_pattern": "[A-z][A-z0-9_]+",
     "keyword_pattern": "[A-z][0-9A-z_]+"
+  },
+  "markup": {
+    "enable_markup": false
```

Comment:
This prevents `cmake-format` from reformatting comments.

ref: https://cmake-format.readthedocs.io/en/latest/configuration.html